test/csuite/truncated_log/main.c

```diff
@@ -164,10 +171,14 @@ fill_db(void)
     max_key = min_key * 2;
     first = true;
     for (i = 0; i < max_key; ++i) {
-        testutil_check(__wt_snprintf(k, sizeof(k), "key%03d", (int)i));
+        if (use_columns)
+            cursor->set_key(cursor, i + 1);
+        else {
+            testutil_check(__wt_snprintf(k, sizeof(k), "key%03d", (int)i));
+            cursor->set_key(cursor, k);
```

Comment:
The `%03d` is worrisome -- I didn't go look, but just confirming that "03" is safe for the maximum key.

Casting to an `int` is kind of ugly, too, might be worth making the printf type correct.

Reply:
The maximum key is apparently 438, so it is, but nothing bad happens if the keys get larger, other than they stop appending at the end of the tree and start inserting in the middle. Which on the whole it might actually be better to exercise, but that's for some other time I think.
I fixed the printf (and the adjacent one for the value).